### PR TITLE
terminal: implement whatis command

### PR DESCRIPTION
**Pull request by aarzilli**
```
terminal: implement whatis command

Implements #596

```

### CODE REVIEW

File: pkg/terminal/command.go [hunk 1]
```diff
@@ -141,6 +142,9 @@ Called with more arguments it will execute a command on the specified goroutine.
 	[goroutine <n>] [frame <m>] print <expression>
 
 See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/expr.md for a description of supported expressions.`},
+		{aliases: []string{"whatis"}, allowedPrefixes: scopePrefix, cmdFn: whatisCommand, helpMsg: `Prints type of an expression.
+		
+		whatis <expression>.`},
```
Comment (derekparker) [inline pkg/terminal/command.go]: Technically this won't eval an entire expression, let's just put `<var name>` or something like that instead.

Comment (aarzilli) [inline pkg/terminal/command.go]: it's actually the same as print, it will evaluate any expression.

Comment (derekparker) [reply pkg/terminal/command.go]: Ah, I see that now, seemed when I looked (and tested) before other args were getting ignored. Either way, this seems fine.

**Comment by derekparker** [comment]
@aarzilli can you regen the docs for this PR and I'll merge.

**Comment by aarzilli** [comment]
Done.

**Pull request merged**